tests/processes/ProcessingTest.py

```diff
@@ -816,3 +818,50 @@ def test_autoapply_override(self):
                                       self.section, self.log_printer, {}, [],
                                       console_printer=self.console_printer)
         self.assertEqual(newres, [])
+
+
+class ESLintBear_apply_patches(unittest.TestCase):
```

Comment:
1. --> `ESLintBearApplyPatches`, don't mix casing conventions ;)
2. Also please append `Test` to the class name, I think it's not picked up otherwise^^

Reply:
EDIT:
Seems the test is picked up, anyway, please adapt to our naming scheme ;)